src/main/java/org/opensearch/searchrelevance/metrics/EvaluationMetrics.java

```diff
@@ -38,14 +32,16 @@ public static Map<String, String> calculateEvaluationMetrics(List<String> docIds
 
         double coverage = totalCount > 0 ? Math.round((double) totalDocsWithScores / totalCount * 100.0) / 100.0 : 0.0;
 
-        // TODO: it's not guarantee that each docId will have its score, especially for UBI data.
-        // Need to define a reliable rate. say, coverage > 80%, then the results become reliable
-        currSearchConfigMetrics.put("coverage", String.valueOf(coverage));
+        // docIds that don't have a score will be assumed irrelevant with a score of 0
+        // Coverage is the metric of what percentage of docIds per query have score
+        currSearchConfigMetrics.put("Coverage@" + k, String.valueOf(coverage));
```

Comment:
nit: we need to refactor later this direct string concatenation to String.format,